pyanaconda/ui/tui/spokes/langsupport.py

```diff
@@ -122,7 +123,7 @@ def input(self, args, key):
             self.app.switch_screen(self, None)
             return True
         else:
-            return key
+            return NormalTUISpoke.input(self, args, key)
```

Comment:
Isn't there a way how to solve this globally? I mean in some base class or similar place.